src/main/java/org/jenkinsci/plugins/tokenmacro/impl/LogRegExMacro.java

```diff
@@ -50,15 +54,21 @@ public String readLogFile(File file) throws IOException
         // Assume default encoding and text files
         String line;
         Pattern pattern = Pattern.compile(regex);
-        BufferedReader reader = new BufferedReader(new FileReader(file));
-        while ((line = reader.readLine()) != null)
-        {
-            Matcher matcher = pattern.matcher(line);
-            if (matcher.find())
+        
+        BufferedReader reader = new BufferedReader(
+                new InputStreamReader(new FileInputStream(file), StandardCharsets.UTF_8));
```

Comment:
> StandardCharsets only available in JDK 1.7 - this should have failed to compiled no?

It's being compiled as a Java6-compatible byte code by JDK7. BTW we may need to support JDK6 as well.

> Are you sure the log is in UTF-8 and not the platform default?

No. Needs to be investigated